### PR TITLE
update configs file

### PR DESCRIPTION
**Pull request by ylil93**
- add comment explaining WHITELIST_URL
- add test to make sure config files match

### CODE REVIEW

File: compatibility_server/configs.py [hunk 1]
```diff
@@ -1,187 +0,0 @@
-# Copyright 2018 Google LLC
```
Comment (liyanhui1228) [inline compatibility_server/configs.py]: The compatibility server is deployed using docker, and it won't be able to find the symlinks without uploading the ../compatibility_lib/ stuff with the same directory structure unfortunately :(

Comment (ylil93) [reply compatibility_server/configs.py]: That's a good point - what if compatibility_lib/compatibility_lib/configs.py is the symlink file instead?

Comment (liyanhui1228) [reply compatibility_server/configs.py]: That's the same, releasing the compatibility_lib to PyPI won't include the files in compatibility_server.

Comment (ylil93) [reply compatibility_server/configs.py]: hmm I guess they're pretty isolated from each other

Maybe I can add a test to catch any differences instead.

Comment (liyanhui1228) [reply compatibility_server/configs.py]: SGTM.

Comment (ylil93) [reply compatibility_server/configs.py]: done

**Pull request merged**